### ISSUE TITLE
Can we access pprof api?

### ISSUE THREAD

**Issue by cloverrose** [yahoojapan/ngtd#14]
pprof is imported in main.go [L22](https://github.com/yahoojapan/ngtd/blob/master/cmd/ngtd/main.go#L22).
So I want to access pprof api via http request.
But I cannot access it now.

In my knowledge we need call http.ListenAndServe, such that

```diff
 func main() {
+	go func() {
+		log.Println(http.ListenAndServe("0.0.0.0:PORT", nil))
+	}()
```

Is it right ? Or ngtd already expose pprof http api ?

**Comment by cloverrose** [comment]
Does anyone know it?

**Comment by kpango** [comment]
As for import of pprof, I think that it probably forgot to erase while debugging, so we will not provide pprof.
Enabling pprof is not good for performance, so I think that we should erase or comment out it.
If you want to use pprof you should debug yourself.

**Comment by kpango** [comment]
resolved by #23
